membership_file/tests/tests_user.py

```diff
@@ -57,6 +57,7 @@ def test_field_label(self):
 ################################################################
 
 # Tests the editing feature of a user's own membership info.
+@override_settings(MEMBERSHIP_FILE_EXPORT_PATH=None)
```

Comment:
why not test the export functionality instead?

Reply:
That's done in a different (and more general) testcase in `tests_export.py`